modules/ROOT/pages/intro-platform-hosting.adoc

```diff
@@ -0,0 +1,146 @@
++
+CloudHub also provides high-availability, clustering and failover of your APIs and Mule applications and load balances traffic to them. See xref:runtime-manager::cloudhub.adoc[CloudHub] for more information.
++
+CloudHub can only be managed by the Cloud control plane in the same environment (US Cloud, EU Cloud, or GovCloud).
+
+* Runtime Fabric
++
+Runtime Fabric is a container service that enables you to run Mule applications and API gateways within a data center or 3rd-party cloud environment that you control and manage. Runtime Fabric can be installed on a set of physical servers, virtual machines, or within Amazon Web Services and Microsoft Azure.
++
+Runtime Fabric comes bundled with technology such as Docker and Kubernetes to offer benefits such as high-availability, failover, clustering, and load balancing. See xref:runtime-fabric::/latest/index.adoc[Anypoint Runtime Fabric Overview] for more information.
++
+Runtime Fabric can only be managed by Cloud control planes (US Cloud, EU Cloud). MuleSoft Government Cloud and Private Cloud Edition do not support Runtime Fabric.
+
+* Standalone Runtimes
++
+The standalone option enables you to host Mule runtime engine server and related services in an environment that you manage. Using standalone runtimes, the Mule Runtime Server can run on a physical server, a virtual machine, or within a 3rd-party cloud installation like Amazon Web Services or Microsoft Azure.
++
+When using standalone runtimes, you are also responsible for providing the framework for high-availability, failover, clustering and load balancing. See xref:mule-runtime::latest/mule-standalone[Run Mule Runtime Engine On-Premises]
++
+Standalone runtimes can be managed by Cloud control plans (US Cloud, EU Cloud) or a customer-hosted control plane (PCE). MuleSoft Government Cloud does not support standalone runtimes.
+
+=== Support Matrix for Runtime Plane Hosting Options
+
+Not all Anypoint Platform components are supported in each of the Runtime Plane hosting options:
+
+[%header,cols="4*a"]
+|===
+| Anypoint Platform Component | CloudHub | Runtime Fabric | Standalone
+| Mule runtime engine | Y | Y | Y
+| Anypoint MQ | Y | N | N
+| Anypoint Object Store | Y | N | N
+| Connectors | Y | Y | Y
+|===
+
+
+* Anypoint MQ and Anypoint Object Store
++
+Runtime Fabric and standalone runtimes do not support Anypoint MQ or Anypoint Object Store. Instead, Runtime Fabric and standalone runtimes support Mule clustering. On Runtime Fabric, multiple replicas of the same application can share objects. On standalone runtimes, multiple Mule runtime engines can share objects with each other.
++
+See xref:mule-runtime::choosing-the-right-clustering-topology[Cluster Runtime Instances]
+
+* Connector Support
++
+Connectors that retrieve and manipulate files on the file system are not supported, for example the file connector. Mule applications deployed to Runtime Fabric do not have access to the file system of the virtual machine nor the network mount. 
```

Comment:
```suggestion
Connectors that retrieve and manipulate files on the file system are not supported, for example, Anypoint Connector for File (File Connector). Mule applications deployed to Runtime Fabric do not have access to the file system of the virtual machine or the network mount. 
```

Reply:
Resolved. Thanks for letting me know the official name of that connector!